module/skill/QuinticWalk/README.md

```diff
@@ -0,0 +1,28 @@
+# QuinticWalk
+
+## Description
+
+Open loop walk engine that uses quintic splines to create trajectories.
+
+## Usage
+
+Include this module to allow the robot to walk.
+
+## Consumes
+
+- `message::skill::Walk` A Task requesting to walk, containing a vector with the desired velocity target.
+
+## Emits
+
+- `message::behaviour::state::Stability` to update the stability when starting and finishing getting up.
```

Comment:
Still talking about getting up.
This module emits stability to say when it's walking (dynamically stable) and standing (standing)

Reply:
done